azure-quantum/azure/quantum/job/job.py

```diff
@@ -151,12 +151,193 @@ def get_results(self, timeout_secs: float = DEFAULT_TIMEOUT):
                     return {histogram_values[i]: histogram_values[i + 1] for i in range(0, len(histogram_values), 2)}
                 else: 
                     raise f"\"Histogram\" array has invalid format. Even number of items is expected."
+            elif self.details.output_data_format == "microsoft.quantum-results.v2":
+                if "DataFormat" not in results or results["DataFormat"] != "microsoft.quantum-results.v2":
+                    raise f"\"DataFormat\" was expected to be \"microsoft.quantum-results.v2\" in the Job results for \"{self.details.output_data_format}\" output format."
```

Comment:
I know this is just copy-paste issue, but we really should get away from `raise <string>` as it's no longer supported in Python 3. Could you please replace these occurrences with `raise ValueError(<string>)` in this file?